ui/scss/component/_claim-search.scss

```diff
@@ -27,6 +27,23 @@
   }
 }
 
+// UPDATE: Add style for checkbox
+.claim-search__checkbox {
+    padding-left: 45%;
+    padding-top: 7%;
```

Comment:
These %s feel a bit like magic numbers. does var(--spacing-s) or -xs -m -l etc not apply here?

Reply:
I'm still new with CSS. We can try what you suggest.